rpc/types_transaction.go

```diff
@@ -211,6 +228,10 @@ func unmarshalTxn(t interface{}) (Transaction, error) {
 		case TransactionType_Declare:
 
 			switch TransactionType(casted["version"].(string)) {
+			case "0x0":
```

Comment:
Wouldn't it be better to use `TransactionVersion` type variables here instead of writing the string yourself?